release-notes/opensearch-alerting.release-notes-1.2.4.0.md

```diff
@@ -0,0 +1,7 @@
+## Version 1.2.4.0 2022-01-18
+
+Compatible with OpenSearch 1.2.0
```

Comment:
I forget, is it compatible with `OpenSearch 1.2.0` or should this be `1.2.4` due to the strong coupling in the current distributions?

Reply:
Good point, I updated it to 1.2.4. Opensearch checks the minor version as well, so it wouldn't be compatible